### PR TITLE
k3s version v1.21.4+k3s1 (3e250fdb) go version go1.16.6 does not crea…

### PR DESCRIPTION
**Pull request by ProjectInitiative**
…te /server/db/snapshots, instead it is now located at /var/lib/rancher/k3s/server/db

### CODE REVIEW

**Comment by ProjectInitiative** [comment]
Created on wrong branch

**Comment by ProjectInitiative** [comment]
After re-reading the contributing guidelines, unclear where to request changes to none versioned rancher docs (i.e. k3s)

**Comment by btat** [comment]
Hi @ProjectInitiative, the info mentioned in the guidelines are not applicable to K3s and RKE, which do not have versioned documentation. What you have now is good as it is.

I will update the PR template so it's more clear.

**Comment by ProjectInitiative** [comment]
> Hi @ProjectInitiative, the info mentioned in the guidelines are not applicable to K3s and RKE, which do not have versioned documentation. What you have now is good as it is.
> 
> I will update the PR template so it's more clear.

I appreciate the information! Thank you for the update.

**Comment by brandond** [comment]
Yeah, this should have been documented as `$DATADIR/server/db/FOO`, as the /var/lib/rancher/k3s prefix can be changed by setting the --data-dir flag. Maybe just call that out as well?

File: content/k3s/latest/en/backup-restore/_index.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ In this section, you'll learn how to create backups of the K3s cluster data and
 
 Snapshots are enabled by default.
 
-The snapshot directory defaults to `/server/db/snapshots`.
+The snapshot directory defaults to `${data-dir}/server/db/snapshots`. Note that the `/var/lib/rancher/k3s` prefix can be changed by setting the `--data-dir` flag. This prefix is currently omitted from the documented paths.
```
Comment (brandond) [inline content/k3s/latest/en/backup-restore/_index.md:33]: If this PR adds it to both the places it was missing from, we can probably drop that bit.

**Review by brandond — CHANGES_REQUESTED** [review]

File: content/k3s/latest/en/backup-restore/_index.md (hunk 1 above)
Comment (jtravee) [reply content/k3s/latest/en/backup-restore/_index.md]: Removed phrase, thanks!

File: content/k3s/latest/en/backup-restore/_index.md [hunk 2]
```diff
@@ -30,13 +30,13 @@ In this section, you'll learn how to create backups of the K3s cluster data and
 
 Snapshots are enabled by default.
 
-The snapshot directory defaults to `/server/db/snapshots`.
+The snapshot directory defaults to `${data-dir}/server/db/snapshots`. Note that the `/var/lib/rancher/k3s` prefix can be changed by setting the `--data-dir` flag.
```
Comment (brandond) [inline content/k3s/latest/en/backup-restore/_index.md:33]:
```suggestion
The snapshot directory defaults to `${data-dir}/server/db/snapshots`. The data-dir value defaults to `/var/lib/rancher/k3s`, and can be changed by setting the `--data-dir` flag.
```

**Review by brandond — CHANGES_REQUESTED** [review]
One last nit on the wording. Thanks for taking the time to get this straightened out!

File: content/k3s/latest/en/backup-restore/_index.md (hunk 2 above)
Comment (jtravee) [reply content/k3s/latest/en/backup-restore/_index.md]: Thanks as always for the feedback!

**Review by brandond — APPROVED** [review]

**Pull request merged**